src/Apps/Artist/Routes/AuctionResults/AuctionResultsFilterContext.tsx

```diff
@@ -29,6 +29,11 @@ export interface AuctionResultsFilterContextProps {
   setFilter: (name: keyof AuctionResultsFilters, value: any) => void
   unsetFilter: (name: keyof AuctionResultsFilters) => void
   onAuctionResultClick?: (index: number) => void
+  onFilterClick?: (
```

Comment:
The `onFilterClick` ended up not being necessary for event tracking (see https://github.com/artsy/reaction/pull/3175) but I kept it here for future use though it doesn't have anything to do with instrumentation